src/components/AppStyle.tsx

```diff
@@ -256,6 +261,18 @@ const CssContainer = styled.div`
  *   - children: All the children that should be rendered with the applied styling.
  */
 export default function AppStyle(props: PropsWithChildren<AppStyleProps>): ReactElement {
+  // Provide a ref for modals to use as a container. This allows them to float over
+  // other elements in the app, and lets us use the <Modal> component from Ant instead
+  // of the static modal API through Ant.App, which is less flexible.
+  const modalContainerRef = useRef<HTMLDivElement>(null);
+
+  // Force a rerender so the context provider updates with the new div ref; otherwise
+  // the value will always be null.
+  const [, setForceRerender] = useState(false);
+  useEffect(() => {
+    setForceRerender(true);
+  }, []);
+
   return (
     <CssContainer className={props.className}>
       <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
curious why this is here and not in your html file. same with the font import above

Reply:
I wanted to have the font family variables all live in one place for DRY reasons, though I know it's non-standard... This may also help if we ever want to embed the viewer as a component somewhere, though there's currently no plans to do so. Do you think this is worth bumping to `index.html`?